src/main/java/frc/robot/commands/intake/RunIntakeReversed.java

```diff
@@ -0,0 +1,44 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.RobotConfig;
+import frc.robot.subsystems.intake.Intake;
+
+public class RunIntakeReversed extends Command {
+  private final Intake m_intake;
+
+  /**
+   * Creates a new RunIntake command, which runs the roller motor on the intake subsystem to intake
+   * a note
+   *
+   * @param intake The subsystem used by this command.
+   */
+  public RunIntakeReversed(Intake intake) {
```

Comment:
Instead of making a whole new class for this, surely you could have just made a single "RunIntake" that accepted a boolean for direction?

Reply:
I covered this with Andrew earlier, but to avoid retesting and another PR that needs to be approved onto dev, it's probably fine to merge it in and fix it in the next PR to main.